.github/workflows/pre-commit.yml

```diff
@@ -1,14 +0,0 @@
-name: pre-commit
```

Comment:
If you remove this, maybe remove the pre-commit config file as well (`.pre-commit-config.yaml`)

Reply:
good catch thanks